tests/kernels/test_prefix_prefill.py

```diff
@@ -39,20 +46,20 @@ def test_contexted_kv_attention(
                         num_heads,
                         head_size,
                         dtype=dtype,
-                        device='cuda')
+                        device=device)
```

Comment:
Can we just add `torch.set_default_device("cuda")` at the beginning of the function body? Same for other test functions.